pages/holiday-hackdays.vue

```diff
@@ -0,0 +1,346 @@
+<template>
+  <div>
+    <transition name="fade">
+      <main class="py-8" v-if="isMounted">
+        <section class="md:py-6 px-4 flex flex-col items-center">
+          <a href="/">
+            <img class="w-12 sm:w-16" :src="SiteLogo" alt="" />
+          </a>
+          <div
+            class="px-1 mt-12 h-36 md:h-52 w-full max-w-screen-lg rounded-lg"
+          >
+            <div class="relative flex h-full w-full">
+              <a
+                href="https://supabase.com"
```

Comment:
```suggestion
                href="https://supabase.com/blog/2021/12/03/supabase-holiday-hackdays-hackathon"
```

Reply:
Yup was going to ask you about the Hackdays' link. 

Only 2 cateogories for this round?
```
Most impressive usage of the Realtime API features
Most fun / holiday themed project
```

Reply:
Correct, only two. Five is quite tough to review and judge for, so trying it with two this time around.